babel/messages/catalog.py

```diff
@@ -93,10 +93,10 @@ def __init__(self, id, string=u'', locations=(), flags=(), auto_comments=(),
                        PO file, if any
         :param context: the message context
         """
-        self.id = id #: The message ID
```

Comment:
hm, would be interesting to know what they're used for here. E.g. in this case the id param already has a documentation.